src/app-layout/__tests__/common.test.tsx

```diff
@@ -142,10 +144,10 @@ describeEachAppLayout({ themes: ['classic', 'refresh'] }, ({ size }) => {
           expect(landmarks).toHaveLength(2);
           const toggleElement = findToggle(wrapper).getElement();
 
-          if (landmarks[0].getElement().contains(toggleElement)) {
+          if (theme !== 'refresh-toolbar' && landmarks[0].getElement().contains(toggleElement)) {
             expect(landmarks[0].getElement()).toHaveAttribute('aria-hidden', 'true');
             expect(landmarks[1].getElement()).toHaveAttribute('aria-hidden', 'false');
-          } else {
+          } else if (theme !== 'refresh-toolbar') {
             expect(landmarks[1].getElement()).toContainElement(toggleElement);
             expect(landmarks[1].getElement()).toHaveAttribute('aria-hidden', 'true');
             expect(landmarks[0].getElement()).toHaveAttribute('aria-hidden', 'false');
```

Comment:
Can we bubble up this repeating part

```js
if(theme === 'refresh-toolbar') {
   // assertions for the toolbar design
} else {
   // current assertions
}
```

I think it would be easier to follow the expected state in this format